test/tools/libtesteth/TestHelper.cpp

```diff
@@ -242,23 +244,69 @@ void replaceLLLinState(json_spirit::mObject& _o)
 	}
 }
 
-std::vector<boost::filesystem::path> getJsonFiles(boost::filesystem::path const& _dirPath, std::string const& _particularFile)
+std::vector<boost::filesystem::path> getFiles(boost::filesystem::path const& _dirPath, std::set<string> const _extentionMask, std::string const& _particularFile)
 {
-	vector<boost::filesystem::path> jsonFiles;
-	if (!_particularFile.empty())
+	vector<boost::filesystem::path> files;
+	for(auto const& ext: _extentionMask)
 	{
-		boost::filesystem::path file = _dirPath / (_particularFile + ".json");
-		if (boost::filesystem::exists(file))
-			jsonFiles.push_back(file);
+		if (!_particularFile.empty())
+		{
+			boost::filesystem::path file = _dirPath / (_particularFile + ext);
+			if (boost::filesystem::exists(file))
+				files.push_back(file);
+		}
+		else
+		{
+			using Bdit = boost::filesystem::directory_iterator;
+			for (Bdit it(_dirPath); it != Bdit(); ++it)
+			{
+				if (boost::filesystem::is_regular_file(it->path()) && it->path().extension() == ext)
+					files.push_back(it->path());
+			}
+		}
 	}
-	else
+	return files;
+}
+
+json_spirit::mValue convertYamlNodeToJson(YAML::Node _node)
+{
+	if (_node.IsNull())
+		return json_spirit::mObject();
+
+	if (_node.IsScalar())
 	{
-		using Bdit = boost::filesystem::directory_iterator;
-		for (Bdit it(_dirPath); it != Bdit(); ++it)
-			if (boost::filesystem::is_regular_file(it->path()) && it->path().extension() == ".json")
-					jsonFiles.push_back(it->path());
+		json_spirit::mValue value;
+		if (_node.Tag() == "tag:yaml.org,2002:int")
+			value = _node.as<int>();
```

Comment:
You can simplify it by `if (_node.Tag() == "tag:yaml.org,2002:int") return _node.as<int>();`.